sr_port/zshow_stack.c

```diff
@@ -30,7 +30,7 @@
 #define ZINTR_FRAME		"    ($ZINTERRUPT) "
 #define DEVERR_FRAME		"    (Device Error)"
 #define DIR_MODE_MESS		"    (Direct mode) "
-#define CALL_IN_BASE_FRAME	"    (Call-In Level Entry)"
+#define CALL_IN_BASE_FRAME	"    " CALL_IN_M_ENTRYREF
```

Comment:
CALL_IN_M_ENTRYREF is a C macro that evaluates to "(Call-In)". It is structured this way because the same string is used in gtmci_ch.c so using a macro avoids string literal duplication.